ods_ci/tasks/Resources/RHODS_OLM/install/install.robot

```diff
@@ -18,7 +18,7 @@ Installing RHODS Operator ${image_url}
   IF  ${is_codeflare_managed}    Installing CodeFlare Operator
 
 RHODS Operator Should Be installed
-  Verify RHODS Installation
+  Verify RHODS Installation    ${cluster_type}
```

Comment:
the is not needed cluster type is global variable

Reply:
Removed